cardano-tx-generator/src/Cardano/Benchmarking/GeneratorTx.hs

```diff
@@ -241,23 +241,25 @@ runBenchmark
                        traceSubmit
                        traceN2N
                        connectClient
+                       "UnknownThreadLabel"
```

Comment:
By the way, can we think of a better thread label here?

Reply:
The thread labels are set in the JSON scripting language.
This path is called from the CLI parser (but not only).
That function is also deprecated with a prama.